Common/AnatupleProducer.py

```diff
@@ -0,0 +1,185 @@
+import ROOT
+import numpy as np 
+import Common.BaselineSelection as Baseline
+import Common.Utilities as Utilities
+import Common.ReportTools as ReportTools
+
+def DefineAndAppend(df, varToDefine, varToCall, colToSave):
+    df = df.Define(f"{varToDefine}", f"{varToCall}")
+    colToSave.append(varToDefine)
+    return df 
+
+def createAnatuple(inFile, outFile, period, sample, X_mass, snapshotOptions, isData=0):
+    Baseline.Initialize(True, True)
+    df = ROOT.RDataFrame("Events", inFile) 
+    df = df.Range(100)
+    df = df.Define("sample", f"static_cast<int>(SampleType::{sample})")
+    df = df.Define("period", f"static_cast<int>(Period::Run{period})") 
+    df = df.Define("X_mass", f"static_cast<int>({X_mass})")
+    
+    df = df.Define("GenPart_daughters", "GetDaughters(GenPart_genPartIdxMother)")
+    df = Baseline.RecoLeptonsSelection(df)
+    df = Baseline.RecoJetAcceptance(df)
+    df = Baseline.RecoHttCandidateSelection(df)
+    df = Baseline.RecoJetSelection(df) 
+    df = Baseline.DefineHbbCand(df)  
+    df = df.Define("channelId","static_cast<int>(httCand.channel())")
+    df = df.Define("is_data", f"{isData}") 
+    df = df.Define("lumi", "luminosityBlock")
```

Comment:
maybe for names which are direct copies of nanoAOD branches we can keep nanoAOD naming convention. What do you think?